Specs/add_line_numbers.rb

```diff
@@ -0,0 +1,17 @@
+def puzzle_2
+  chapter2 = File.read('add_line_numbers.txt')
+  output = add_line_numbers(chapter2)
+  print output
+end
+
+def add_line_numbers(chapter2)
```

Comment:
This method should work for any kid of multiline text. It does not have to be a chapter.

```suggestion
def add_line_numbers(lines)
```